zipkin-common/src/main/scala/com/twitter/zipkin/common/Endpoint.scala

```diff
@@ -63,5 +63,5 @@ case class Endpoint(ipv4: Int, port: Short, serviceName: String) extends Ordered
 
 object Endpoint {
```

Comment:
higher in the file, on the serviceName field, I'd add a note that "" implies the span won't be indexed. (or similar succinct thing)